src/command-line/install.js

```diff
@@ -38,45 +38,54 @@ program
 			const packagesParent = path.dirname(packagesPath);
 			const packagesConfig = path.join(packagesParent, "package.json");
 
-			// Create node_modules folder, otherwise npm will start walking upwards to find one
+			// Create node_modules folder, otherwise yarn will start walking upwards
+			// to find one
 			fsextra.ensureDirSync(packagesPath);
 
-			// Create package.json with private set to true to avoid npm warnings, if
-			// it doesn't exist already
+			// Create package.json with private set to true, if it doesn't exist
+			// already
 			if (!fs.existsSync(packagesConfig)) {
 				fs.writeFileSync(packagesConfig, JSON.stringify({
 					private: true,
 					description: "Packages for The Lounge. All packages in node_modules directory will be automatically loaded.",
 				}, null, "\t"));
 			}
 
-			const npm = child.spawn(
-				process.platform === "win32" ? "npm.cmd" : "npm",
+			let success = false;
+			const yarn = child.spawn(
+				process.platform === "win32" ? "yarn.cmd" : "yarn",
 				[
-					"install",
+					"add",
+					"--json",
+					"--exact",
 					"--production",
-					"--save",
-					"--save-exact",
-					"--no-bin-links",
-					"--no-package-lock",
-					"--no-progress",
-					"--prefix",
+					"--ignore-scripts",
+					"--non-interactive",
+					"--cwd",
 					packagesParent,
 					packageName,
-				],
-				{
-					// This is the same as `"inherit"` except `process.stdout` is ignored
-					stdio: [process.stdin, "ignore", process.stderr],
-				}
+				]
 			);
 
-			npm.on("error", (e) => {
+			yarn.stdout.on("data", (data) => {
+				data.toString().trim().split("\n").forEach((line) => {
+					line = JSON.parse(line);
+
+					if (line.type === "step") {
+						log.info(line.data.message);
```

Comment:
This now displays:

<img width="658" alt="screen shot 2018-01-18 at 01 36 47" src="https://user-images.githubusercontent.com/113730/35084082-3500e8c0-fbf0-11e7-9a53-d3bca3b6a1f4.png">

i.e. lines 3 to 6 are new. I don't think they add any value to the user (it is all summarized in the second line, ours) so we should silence them.
In fact, I think first line is not very useful either, we might want to get rid of it too. When it comes to fetching metadata, it only matters to know that the package is not a TL package, which we display as an error.

Reply:
Removed.